SensorFusion_src/parse.c

```diff
@@ -1 +1,76 @@
-// the code for parsing the input file and converting it into a 1Xn array goes here
\ No newline at end of file
+/*******************************************************************************************/
+/* Author : Shyam Bhuptani 
+/* Function : Parse 
+/* Application : This function converts the input csv file into a sensor reading1-D array
+/* this parse file is for tag 1 which only works fine for one time value
+/* here the number of sensor is fixed which is 3
+/* Input : csv file of data  with 3 fields Time(24hr clock) , sensor_name, value
+/* Output : sensor reading array of size N (double)
+*/
+
+/*Including dependancies*/
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>	
+
+#include "../include/parse.h"	 
+//Number of sensors are fixed
+#define num_of_sensor 3
+
+/*Function declaration which returns a 2D array */
+double *Parse(char file_name[])

```

Comment:
Why does this function name start with a capital letter?

Reply:
Parse has been removed as it is fully created by amuleen.
this was just an interim development which worked for all inputs at a single time.